plan/logical_plan_builder.go

```diff
@@ -911,54 +910,59 @@ type ApplyConditionChecker struct {
 	All       bool
 }
 
-func (b *planBuilder) buildApply(p, inner LogicalPlan, schema expression.Schema, checker *ApplyConditionChecker) LogicalPlan {
+func (b *planBuilder) buildApply(outerPlan, innerPlan LogicalPlan, checker *ApplyConditionChecker) LogicalPlan {
```

Comment:
I think that add simple comment for innerPlan and outerPlan will be more readable.